Sources/TarkaUI/Components/DraggableCard/TUIDraggableCard.swift

```diff
@@ -1,69 +1,104 @@
 //
 //  TUIDraggableCard.swift
-//  
+//
 //
 //  Created by Arvindh Sukumar on 08/05/23.
 //
 
 import SwiftUI
 
-/// A view that represents a draggable card with an icon button and content.
+/// A view that represents a draggable card with an icon button, title and toggle.
 ///
-/// The `TUIDraggableCard` view is a container view that displays an icon button and content. The icon button can be used to trigger a drag gesture to move the card around. The content can be any SwiftUI view.
+/// The `TUIDraggableCard` view is a container view that displays an icon button  title and toggle.
+///  The icon button can be used to trigger a drag gesture to move the card around. The content can be any SwiftUI view.
 ///
 /// Example usage:
 ///
-///     TUIDraggableCard {
-///       HStack {
-///         Text("Description")
-///           .font(.heading6)
-///           .foregroundColor(Color.inputText)
-///
-///         Spacer()
-///       }
-///     }
+///     TUIDraggableCard("Description") { }
+///       .toggleAction(true) { }
 ///
 /// - Parameters:
-///   - content: A closure that returns the content to display in the card.
+///   - title: It's used to display the title of the view
+///   - action: It will tigger when user tap and hold for 0.5 seconds
+///
+/// - Returns: A view that represents a draggable card with an icon button, title and toggle.
 ///
-/// - Returns: A view that represents a draggable card with an icon button and content.
-public struct TUIDraggableCard<Content>: View where Content: View {
-  var content: () -> Content
+public struct TUIDraggableCard: View {
   
-  /// Creates a draggable card with an icon button and content.
-  ///
-  /// - Parameters:
-  ///   - content: A closure that returns the content to display in the card.
-  ///
-  public init(@ViewBuilder content: @escaping () -> Content) {
-    self.content = content
+  private let title: String
+  private var action: () -> Void
+  
+  private var toggleSelected: Bool
```

Comment:
pls add `is` as prefix for consistency
```suggestion
  private var isToggleSelected: Bool
```